packages/afl-persistent/afl-persistent.1.2/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+maintainer: "stephen.dolan@cl.cam.ac.uk"
+authors: ["Stephen Dolan"]
+homepage: "https://github.com/stedolan/ocaml-afl-persistent"
+bug-reports: "https://github.com/stedolan/ocaml-afl-persistent/issues"
+dev-repo: "https://github.com/stedolan/ocaml-afl-persistent.git"
+license: "MIT"
+build: [[ "./build.sh" ]]
+depends: [
+  "ocamlfind"
+  "base-unix"
+]
+post-messages: [
+"afl-persistent is installed, but since AFL instrumentation is not available
+with this OCaml compiler, instrumented fuzzing with afl-fuzz won't work.
+
+To use instrumented fuzzing, switch to an OCaml version supporting AFL, such
+as 4.04.0+afl." {success & !afl-available}
```

Comment:
Using a [useful but under-documented feature of opam](https://opam.ocaml.org/doc/manual/dev-manual.html#sec37), it takes them out of `afl-persistent.config` [as generated by the build](https://github.com/stedolan/ocaml-afl-persistent/blob/master/build.sh#L31-L35).